WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/BarViewModel.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using ComparisonEngineUI.Views;
+using Xamarin.Forms;
+using ComparisonEngineUI.Models;
+using ComparisonEngineUI.Data;
+
+namespace ComparisonEngineUI.ViewModels
+{
+    
+    public class BarViewModel : BaseViewModel
+    {
+        private ListContainer listContainer = ListContainer.Instance;
+        public Bar bar;
+        private string _Name;
+        public string Name
+        {
+            get
+            {
+                return _Name;
+            }
+            set
+            {
+                _Name = value;
+                OnPropertyChanged();
+            }
+        }
+        private string _Location;
+        public string Location
+        {
+            get
+            {
+                return _Location;
+            }
+            set
+            {
+                _Location = value;
+                OnPropertyChanged();
+            }
+        }
+        private List<SpecificPrice> _SpecificPriceList { get; set; }
+        public List<SpecificPrice> SpecificPriceList
+        {
+            get
+            {
+                return _SpecificPriceList;
+            }
+            set
+            {
+                if (value != _SpecificPriceList)
+                {
+                    _SpecificPriceList = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
```

Comment:
and once again, but why have get and set for private now? cuz then it's not consistent with name and location